data/hai/hai news.txt

```diff
@@ -0,0 +1,251 @@
+			"Hai can terraform planets a hundred times faster than we can. I wonder why they didn't do that at Darkwaste?"
+			"Hai have the most amazing holographic technology. One of them put on a backpack holoprojector. When he turned it on, he looked like an elephant. The legs and tail even tracked his movements! I wonder if they can mimic humans too?"
+			"The Hai have an amazing grasp of quantum chromodynamics. It's why their shield generators are so effective."
+			"The alloys Hai use in their ships are way beyond our technology."
+
+# Visual suggestion: young, normal-looking, more female than male
+# or people with animals
+news "cute story in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Tourist"
+	portrait
+		portrait/human00
+		portrait/human01
+		portrait/human11
+		portrait/human18
+		portrait/human35
+		portrait/human55
+		portrait/human66
+		portrait/human69
+		portrait/human107
+		portrait/human114
+		portrait/human123
+		portrait/human128
+	message
+		word
+			"My little daughter loved her stuffed bear back in human space, but now she wants to hug every Hai she sees instead."
+			"These Hai are so fluffy I want to hug one every time I see them!"
+			"I saw the Hai-Human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
+			"I saw a Hai wearing the cutest bonnet the other day. When I get back to human space, I'm definitely getting a pet squirrel."
+			"Do you see that Hai woman carrying her baby? Doesn't it look like a long-tailed teddy-bear? The Hai are so cute."
+
+# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
+news "concerned merchant in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Merchant captain"
+	portrait
+		portrait/human02
+		portrait/human04
+		portrait/human06
+		portrait/human10
+		portrait/human15
+		portrait/human17
+		portrait/human31
+		portrait/human58
+		portrait/human63
+		portrait/human72
+		portrait/human73
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
+			"I took a shortcut through Wah Ki with six ships. Just two seconds later, BAM! half my fleet is dust."
+			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"
+			"These Hai are so creepy. What are they hiding? I can't wait to get back to human space."
+			"The Hai don't like my ship's armory of laser rifles, but I was once boarded by a pirate crew and I'm never letting that happen again! If some overgrown squirrel is going to take away my rifle, it'll have to be from my cold dead hands!"
+			"I met some young Hai pilots. It sounds like they start out as poor as we do."
```

Comment:
This one is already under a section with the same portraits and name. Might as well remove it.

Reply:
Done.